Framework/Algorithms/src/SolidAngle.cpp

```diff
@@ -79,8 +170,8 @@ void SolidAngle::exec() {
     m_MaxSpec = numberOfSpectra - 1;
   }
 
-  API::MatrixWorkspace_sptr outputWS = WorkspaceFactory::Instance().create(
-      inputWS, m_MaxSpec - m_MinSpec + 1, 2, 1);
+  MatrixWorkspace_sptr outputWS =
+      WorkspaceFactory::Instance().create(inputWS, numberOfSpectra, 2, 1);
```

Comment:
I think this breaks the backward compatibility (and the tests). Previously the output was only for the range of spectra requested (all by default).